<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Students - шаблоны документов</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pic/window-icon.png</normaloff>pic/window-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_01">
         <property name="maximumSize">
          <size>
           <width>402</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Инструкция:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_02">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Список студентов:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTextBrowser" name="tb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>402</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Программа &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Students&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; предназначена для ....&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;1.&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; Откройте &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;...&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;  → &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Открыть&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;...&lt;/span&gt;&lt;/p&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; font-size:11pt;&quot; style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;???;&lt;/li&gt;
&lt;li style=&quot; font-size:11pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;???;&lt;/li&gt;
&lt;li style=&quot; font-size:11pt;&quot; style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;???.&lt;/li&gt;&lt;/ul&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;2.&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; ???&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;3.&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; ??? &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;???&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; font-size:11pt;&quot; style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;??? &lt;span style=&quot; font-weight:600;&quot;&gt;???&lt;/span&gt;;&lt;/li&gt;&lt;/ul&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; font-size:11pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;нажмите кнопку &lt;span style=&quot; font-weight:600;&quot;&gt;???&lt;/span&gt;;&lt;/li&gt;&lt;/ul&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; font-size:11pt;&quot; style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;??? &lt;span style=&quot; font-weight:600;&quot;&gt;???&lt;/span&gt;.&lt;/li&gt;&lt;/ul&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;4.&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; Сохраните результат → &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Сохранить&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;...&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_03">
         <property name="maximumSize">
          <size>
           <width>402</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Шаблон документа:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QComboBox" name="cb1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::InsertBeforeCurrent</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>---</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Заявление о направлении на практику</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Рабочий график (план) проведения практики</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Индивидуальное задание на практику</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Инструктаж по технике безопасности</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_04">
         <property name="maximumSize">
          <size>
           <width>402</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Операции с документами:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pb_open_xls">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>48</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Открыть файл XLS...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>pic/excel-icon.png</normaloff>pic/excel-icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pb_save_docx">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>48</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Сохранить DOCX...  </string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>pic/word-icon.png</normaloff>pic/word-icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="pb_save_pdf">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>48</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Сохранить PDF...     </string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>pic/pdf-icon.png</normaloff>pic/pdf-icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="pb_send_email">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>48</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Отправить E-MAIL...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>pic/email-icon.png</normaloff>pic/email-icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1" rowspan="8">
        <widget class="QTableWidget" name="tw">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>30</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>180</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>30</number>
         </attribute>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>4</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>5</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>6</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>7</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>8</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>9</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>10</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>11</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>12</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>13</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>14</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>15</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>16</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>17</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>18</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>19</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>20</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>21</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>22</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>23</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>24</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>25</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>26</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>27</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>28</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>29</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>30</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>31</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>32</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>33</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>34</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>35</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>36</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>37</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>38</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>39</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>40</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>41</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>42</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>43</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>44</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>45</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>46</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>47</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>48</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>49</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>50</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>ФИО</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Курс</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Группа</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Форма обучения</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Телефон</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>e-mail</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Часы</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Начало</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Конец</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
